torchbenchmark/models/clip/__init__.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+from PIL import Image
+import numpy as np
+import cv2
+import torch
+import os
+from ...util.model import BenchmarkModel
+from torchmultimodal.transforms.clip_transform import CLIPTextTransform, CLIPImageTransform
+from torchmultimodal.models.clip.model import clip_vit_b32
+from torchmultimodal.modules.losses.contrastive_loss_with_temperature import (
+    ContrastiveLossWithTemperature,
+)
+from PIL import Image
+import math
+
+
+
+class Model(BenchmarkModel):
+    DEFAULT_EVAL_BSIZE = 32
```

Comment:
not used?

Reply:
so question here is clip typically used in bs=1 regimes? We were surprised to see that was the case for sam for example

cc @cpuhrsch

Reply:
Added support for batch size